solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -59,13 +60,27 @@ export async function testTokenDeployment(): Promise<DeployedContracts> {
   return contracts
 }
 
-export async function randomBeaconDeployment(): Promise<DeployedContracts> {
-  const SortitionPoolStub = await ethers.getContractFactory("SortitionPoolStub")
-  const sortitionPoolStub: SortitionPoolStub = await SortitionPoolStub.deploy()
-
+export async function randomBeaconDeployment(
+  sortitionPoolStub: Contract
```

Comment:
You can make it an optional parameter: `sortitionPoolStub?: Contract`

Reply:
Done in https://github.com/keep-network/keep-core/pull/2684/commits/625a2862af7bad90fb3d5bfd83a8ff861e901892.